preview-src/comment.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from 'react';
+import { useContext, useState, useEffect, useRef } from 'react';
+
+import Markdown from './markdown';
+import { Spaced, nbsp } from './space';
+import { Avatar, AuthorLink } from './user';
+import Timestamp from './timestamp';
+import { Comment } from '../src/common/comment';
+import { PullRequest } from './cache';
+import PullRequestContext from './context';
+import { editIcon, deleteIcon } from './icon';
+
+export type Props = Partial<Comment & PullRequest>;
+
+export function CommentView(comment: Props) {
+	const { id, pullRequestReviewId, canEdit, canDelete, user, author, htmlUrl, createdAt, bodyHTML, body, } = comment;
+	const [ bodyMd, setBodyMd ] = useState(body);
+	const { deleteComment, editComment, pr } = useContext(PullRequestContext);
+	const currentDraft = pr.pendingCommentDrafts && pr.pendingCommentDrafts[id];
+	const [inEditMode, setEditMode] = useState(!!currentDraft);
+	const [showActionBar, setShowActionBar] = useState(false);
+
+	useEffect(() => {
+		if (body !== bodyMd) {
+			setBodyMd(body);
+		}
+	}, [body]);
+
+	if (inEditMode) {
+		return <EditComment id={id}
+			body={currentDraft || body}
+			onCancel={
+				() => setEditMode(false)
+			}
+			onSave={
+				async text => {
+					try {
+						await editComment({ comment: comment as Comment, text });
```

Comment:
Also for the description, the command is different than a normal comment - `pr.edit-description` instead of `pr.edit-comment`. Title is `pr.edit-title`

Reply:
Fixed.